File: 2024/09 - September/07. Nth Natural Number/Solution.cpp
class Solution 
{
  public:
    long long findNth(long long n) 
    {
        long long result = 0, base = 1;
        while (n > 0) 
        {
            result += (n % 9) * base;
            n /= 9;
            base *= 10;
        }
        return result;
    }
};

File: 2024/09 - September/07. Nth Natural Number/Solution.java
class Solution 
{
    long findNth(long n) 
    {
        long res = 0, base = 1;
        while(n > 0)
        {
            res += (n % 9) * base;
            n /= 9;
            base *= 10;
        }
        return res;
    }
}
